src/components/wallets/wallets.tsx

```diff
@@ -76,7 +78,7 @@ export const Wallets = ({
             <WalletCard
               testID={`${testID}_${w.address}`}
               wallet={w}
-              balance={balance[w.address] ?? 0}
+              balance={formatBalanceWithWEI(balance[w.address])}
```

Comment:
here should be Balance type without formatting. Format it into number we can on last component before render

Reply:
Fixed